src/store/actions.js

```diff
@@ -80,19 +80,22 @@ export default {
     txs = orderBy(flatten(txs), ['microTime'], ['desc']);
     commit('addTransactions', recent ? txs.slice(0, limit) : txs);
   },
-
-  async getCurrencies({ state: { nextCurrenciesFetch }, commit }) {
-    if (!nextCurrenciesFetch || nextCurrenciesFetch <= new Date().getTime()) {
+  async getCurrencies({ state: { nextCurrenciesFetch, currencies }, commit }, delay) {
+    if (!nextCurrenciesFetch
+    || Number(((nextCurrenciesFetch - new Date().getTime()) / 1000).toFixed()) < 1
```

Comment:
`new Date().getTime()` can be replaced with `Date.now()`
Did you mean to use `Math.floor/round/ceil` instead of `Number(x.toFixed())`?
But I don't understand change in general, the previous version looks fine.

Reply:
I've changed this condition because when I was testing it on smaller intervals it gave me some small differences in time but I think `Math.floor` will fix that

Reply:
`nextCurrenciesFetch` is a time at some moment + 1 hour, before we had a check `nextCurrenciesFetch < Date.now()` that is clear, you proposing to change it to `nextCurrenciesFetch < Date.now() + 1 second`,  why do we need this second?

Or you mean to compare `nextCurrenciesFetch` and `Date.now()` accurate to the second? Then you missed `Math.abs`. But still I don't see the reason 🤔

Reply:
I want the difference to be less than 1 second because when I run `setInterval` I sometimes get few millisecond difference between `Date.now()` and `nextCurrenciesFetch` and because of that the currencies will not fetch. `Math.abs()` is not needed here because I want to accept all differences that are smaller than 1s.